### PR TITLE
ENDOC-485 add client role instruction to v632

### PR DESCRIPTION
**Pull request by jyunmitch**
Same check-md error, for corrected error in gke-install.md

The Customer Portal page looks like I made a lot of changes but I just switched the order of sections around. I moved the configuration of CP below the configuration of the Keycloak management. Thought that made more sense since the Keycloak part sets up the management of the CP.

### CODE REVIEW

File: vuepress/docs/v6.3.2/tutorials/solution/customer-portal.md [hunk 1]
```diff
@@ -63,6 +63,53 @@ Users who need access to the Customer Portal, beyond subscription and project in
...
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page shown here. Like the App Builder, the left navigation bar is your guide for managing users, groups, and most importantly, roles. Using the RBAC model, define what access users have by the roles and groups they are assigned. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/cp-idmanagement-main.png)
+
+1. **Set up permissions to configure the service.**
+   - [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
+   - Give at least one user the ability to manage the Customer Portal by granting the `cp-admin` role. Assign the `cp-admin` role for the `pn-a71d68dd-166dc0f4-entandodemo-customerportal-server` client. See [Role Assignment in ID Management](../../docs/consume/identity-management.md#authorization) for more details.
+   - Give the generated plugin client permission to manage users. 
+       1. From the left sidebar, go to `Clients` and select client ID `pn-a71d68dd-166dc0f4-entandodemo-customerportal-server`.
```
Comment (nshaw) [inline vuepress/docs/v6.3.2/tutorials/solution/customer-portal.md:76]: This is the 7.0 client id. We need to use the 6.3.2 client id here.

File: vuepress/docs/v6.3.2/tutorials/solution/pda-tutorial.md [hunk 2]
```diff
@@ -52,8 +52,10 @@ Follow the steps below to configure service permissions and connections.
 1. [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
 
 2. [Assign client roles](../../docs/consume/identity-management.md#authorization) to authorize permissions.
-- Select `pn-b0ceabd7-efbd66b6-entando-pda-plugin-server` from `Client Roles`.
-- Select all roles from `Available Roles`.
+- Select `pn-efbd66b6-b0ceabd7-entando-pda-plugin-server` from `Client Roles`.
```
Comment (nshaw) [inline vuepress/docs/v6.3.2/tutorials/solution/pda-tutorial.md:55]: Sorry, I missed this on the last review. This is the 7.0 client id. We need to switch back to the 6.3.2 client id for this tutorial.

File: vuepress/docs/v6.3.2/tutorials/solution/customer-portal.md [hunk 3]
```diff
@@ -63,6 +63,53 @@ Users who need access to the Customer Portal, beyond subscription and project in
...
 
+
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page shown here. Like the App Builder, the left navigation bar is your guide for managing users, groups, and most importantly, roles. Using the RBAC model, define what access users have by the roles and groups they are assigned. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/cp-idmanagement-main.png)
+
+1. **Set up permissions to configure the service.**
+   - [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
+   - Give at least one user the ability to manage the Customer Portal by granting the `cp-admin` role. Assign the `cp-admin` role for the `pn-a71d68dd-166dc0f4-entandodemo-customerportal-server` client. See [Role Assignment in ID Management](../../docs/consume/identity-management.md#authorization) for more details.
```
Comment (nshaw) [inline vuepress/docs/v6.3.2/tutorials/solution/customer-portal.md:74]: Change to the 6.3.2 client id.

Comment (nshaw) [reply vuepress/docs/v6.3.2/tutorials/solution/customer-portal.md:74]: entandodemo-customerportal-server

File: vuepress/docs/v6.3.2/tutorials/solution/pda-tutorial.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/v6.3.2/tutorials/solution/pda-tutorial.md:55]: entando-pda-plugin-server

**Review by nshaw — CHANGES_REQUESTED** [review]

**Review by Lyd1aCla1r3 — APPROVED** [review]

**Review by nshaw — APPROVED** [review]

**Pull request merged**